src/addons/Display/qtDisplay.py

```diff
@@ -273,12 +297,12 @@ def mouseMoveEvent(self, evt):
         # DRAW BOX
         # ZOOM WINDOW
         elif (buttons == QtCore.Qt.RightButton and
-              modifiers == QtCore.Qt.ShiftModifier):
+                      modifiers == QtCore.Qt.ShiftModifier):
             self._zoom_area = True
             self.DrawBox(evt)
         # SELECT AREA
         elif (buttons == QtCore.Qt.LeftButton and
-              modifiers == QtCore.Qt.ShiftModifier):
+                      modifiers == QtCore.Qt.ShiftModifier):

```

Comment:
this is how my editor ( pycharm ) does auto-formatting.
formatting is important, but lets not go overboard?